UPGRADE-4.0.md

```diff
@@ -226,6 +226,10 @@ Finder
 Form
 ----
 
+* The values of the `FormEvents::*` constants have been updated to match the
+  constant names. You should only update your application if you relied on the
+  constant values instead of their names.
+
```

Comment:
This will cause UPGRADE-4.0 in master and 3.4 going out of sync no? Which btw is already happening (https://www.diffchecker.com/5W6GeCnZ).. again :-)

Reply:
So the upgrade entry should be updated in 3.4 too?

Reply:
yes, it should be synchronized